pkg/runtime/kubernetes/environment_create.go

```diff
@@ -14,9 +14,12 @@ import (
 	"k8s.io/apimachinery/pkg/api/resource"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 
```

Comment:
Please remove the new line to merge these two import groups.

Reply:
Can you elaborate on this?

Reply:
Can you elaborate on this ?